dev/index.jsx

```diff
@@ -1,108 +1,44 @@
 import React, { Component } from 'react';
 import ReactDOM from "react-dom";
 import _ from 'lodash';
-import Autosuggest from 'react-autosuggest';
 import MovieDetails from './movieDetails.jsx';
+import MovieSuggestions from './movieSuggestions.jsx';
 
-class Search extends Component {
+class Index extends Component {
 
   constructor(props) {
     super(props);
-    this.state = { data:'', showResults: false, total_results: 0, id: '', term: '' };
-    this.getMovieList = this.getMovieList.bind(this);
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  componentWillMount() {
-    document.addEventListener('click', this.handleClick, false);
-  }
-
-  componentWillUnmount() {
-    document.removeEventListener('click', this.handleClick, false);
-  }
-
-  handleClick(e) {
-    if(e.target.nodeName !== "LI" && e.target.nodeName !== "INPUT") {
-      this.setState({ showResults: false });
-    }
-  }
-
-  getMovieList(value) {
-    var url = 'https://api.themoviedb.org/3/search/movie?query=' + value + '&api_key=989def4a1b101991db8e783b73ee7bd0';
-
-      fetch(url)
-      .then((response) => response.json())
-      .then((responseJson) => {
-          if(responseJson.total_results == 0) {
-            this.setState({ showResults: false, total_results: responseJson.total_results });
-          }
-      this.setState({data: responseJson.results, total_results: responseJson.total_results });
-    })
-  }
-
-  getmovieDetails(value) {
-    var url = 'https://api.themoviedb.org/3/movie/' + parseInt(value) + '?api_key=989def4a1b101991db8e783b73ee7bd0&language=en-US',
-      enteredMovie = value, results;
-
-      fetch(url)
-      .then((resp) => resp.json())
-      .then((responseJson) => {
-        this.setState({
-          title: responseJson.title.toUpperCase(),
-          overview: responseJson.overview,
-          release_date: responseJson.release_date,
-          vote_average: responseJson.vote_average,
-          poster_path: responseJson.poster_path,
-          runtime: responseJson.runtime,
-          revenue: responseJson.revenue
-        });
-      });
-  this.setState({ showResults: false });
-  }
-
-  componentDidMount() {
-    this.getmovieDetails(278927);
+    this.state = {
+      movieId: '',
```

Comment:
This should be set to a default value. You are doing this on the movieDetails component constructor right now (and it's wrong). Just do this here: 
```
this.state = {
  movieId: '278927',
  ...
}
```